_data/examples.json

```diff
@@ -134,15 +166,27 @@
       "title": "Streamgraph"
     },
     {
-      "name": "bar_layered_transparent",
-      "title": "Layered Bar Chart"
+      "name": "area_horizon",
+      "title": "Horizon Graph",
+      "style": "background-size: auto 70%; background-position-y: center;"
+    }
+  ],
+  "Table-based Plots": [
+    {
+      "name": "rect_heatmap",
+      "title": "Table Heatmap",
+      "style": "background-size: auto"
     },
     {
-      "name": "bar_diverging_stacked",
-      "title": "Diverging Stacked Bar Chart"
+      "name": "rect_binned_heatmap",
+      "title": "Table Binned heatmap"
+    },
+    {
+      "name": "circle_github_punchcard",
+      "title": "Table Bubble Plot (Github Punch Card)"
     }
   ],
-  "Window Transform": [
+  "Advanced Calculations": [
```

Comment:
Let's leave this as window transforms until we have more.

Reply:
I think a lot of people who come to the gallery could be novice who do not know what window is.  Naming the section "window" makes it more confusing.

Reply:
Plus, a lot of these examples are using window and calculate to achieve the result, not purely "window".